grammar/vernacextend.mlp

```diff
@@ -89,6 +89,7 @@ EXTEND
   rule:
     [ [ "["; OPT "-"; l = LIST1 args; "]";
         d = deprecation; c = OPT classifier; "->"; "["; e = Pcaml.expr; "]" ->
+        (** XXX Error on nonempty attributes? *)
       let b = <:expr< fun ~{atts} ~{st} -> ( let () = $e$ in st ) >> in
```

Comment:
I'll try adding an atrribute check here later, see if anything breaks.

Reply:
Seems like nothing breaks.

Reply:
@ppedrot told me the `atts` variable is in scope in the expression so nevermind.